raiden/ui/web/src/app/components/token-input/token-input.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, Input } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { MatCheckboxChange } from '@angular/material';
+import { BigNumber } from 'bignumber.js';
+import { amountFromDecimal, amountToDecimal } from '../../utils/amount.converter';
+
+
+@Component({
+    selector: 'app-token-input',
+    templateUrl: './token-input.component.html',
+    styleUrls: ['./token-input.component.css'],
+})
+export class TokenInputComponent {
+
+    @Input() inputType: string;
+    @Input() inputDescription: string;
+
+    inputControl: FormControl = new FormControl(0);
```

Comment:
Please, make this member `private`, and expose the properties you need through getters, no need for users of the component to know about internal workings of it.